ballerina/records.bal

```diff
@@ -128,13 +128,13 @@ public type AuthenticationConfiguration record {|
 # + concurrentConsumers - Number of concurrent consumers
 # + defaultApiTimeout - Default API timeout value (in seconds) for APIs with duration
 # + autoCommit - Enables auto committing offsets
-# + checkCRCS - Check the CRC32 of the records consumed. This ensures that no on-the-wire or on-disk corruption to
-#               the messages occurred. This may add some overhead, and might needed set to `false` if extreme
+# + checkCRCS - Checks the CRC32 of the records consumed. This ensures that no on-the-wire or on-disk corruption to
```

Comment:
```suggestion
# + checkCRCS - Checks the CRC32 of the records consumed. This ensures that no on-the-wire or on-disk corruption occurred
```

Reply:
Added in https://github.com/ballerina-platform/module-ballerinax-kafka/pull/377/commits/d8e897a4fcf01ef0602e9e87794a77441a554f65